plugins/module_utils/ndb/time_machines.py

```diff
@@ -261,3 +264,59 @@ def _build_spec_schedule(self, payload, schedule):
 
         payload["schedule"] = schedule_spec
         return payload, None
+
+    def get_default_data_access_spec(self, override_spec=None):
```

Comment:
get_default_data_access_spec can be called get_default_data_access_management_spec for more relevance